.github/workflows/canary.yml

```diff
@@ -24,6 +24,7 @@ env:
   TF_VAR_aws_access_key_id: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
   TF_VAR_aws_secret_access_key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
   TF_VAR_patch: 'true'
+  CANARY_INTEGRATION_TEST_REGION: us-east-2
```

Comment:
so we're using us-east-2 here and we were using us-west-2 before, is there a reason for that?

Reply:
Not much. Mostly, I am still finding the right balancer when splitting these test into two different regions, maybe three at most though.